tests/test_per_channel.py

```
@@ -0,0 +1,125 @@
+import torch
+
+from inwhale.core.uniform import (
+    AsymmetricUniformQuantizer,
+    PerChannelAsymmetricUniformQuantizer,
+)
+from inwhale.observers.minmax import MinMaxObserver
+from inwhale.rounding.nearest import NearestRounding
+from inwhale.observers.minmax import MinMaxObserver
```

Ruff (0.14.10)

Redefinition of unused `MinMaxObserver` from line 7

Remove definition: `MinMaxObserver`

(F811)